examples/context/theme-detailed-app.js

```diff
@@ -0,0 +1,30 @@
+import ThemeContext, {themes} from './theme-context';
```

Comment:
I don't think we should show a mixed import like this.

Maybe we can import `themes` from a separate `themes.js` file? Or change `theme-context` to remove the default export.